twisted/plugins/dht_plugin.py

```diff
@@ -0,0 +1,126 @@
+"""
+This twistd plugin enables to start Tribler headless using the twistd command.
+"""
+import os
+import signal
+
+from Tribler.Core.Config.tribler_config import TriblerConfig
+from twisted.application.service import MultiService, IServiceMaker
+from twisted.conch import manhole_tap
+from twisted.internet import reactor
+from twisted.plugin import IPlugin
+from twisted.python import usage
+from twisted.python.log import msg
+from zope.interface import implements
+
+from Tribler.Core.Modules.process_checker import ProcessChecker
+from Tribler.Core.Session import Session
+
+
+class Options(usage.Options):
+    optParameters = [
+        ["manhole", "m", 0, "Enable manhole telnet service listening at the specified port", int],
+        ["statedir", "s", None, "Use an alternate statedir", str],
+        ["restapi", "p", 8085, "Use an specific port for the REST API", int],
+        ["ipv8", "d", -1, "Use an alternate port for IPv8", int],
+    ]
+
+
+class DHTServiceMaker(object):
```

Comment:
Two comments:
- I noticed that we are reusing so much of the original Tribler plugin which leads to many different twistd plugins with duplicate code. At one point, we should merge them all together and just provide the Tribler twistd plugin (with additional parameters to run DHT/market etc).
- When should this plugin be used?

Reply:
I used it mostly for testing, but we could also use it to run DHT routers. Since we don't really need it I'm OK with removing it.

Reply:
Ok, can you remove this file then? 👍